packages/EasyErrorHandler/src/Bridge/Laravel/Translator.php

```diff
@@ -14,19 +15,37 @@ final class Translator implements TranslatorInterface
      */
     private $decorated;
 
-    public function __construct(IlluminateTranslatorInterface $decorated)
+    /**
+     * @var string
+     */
+    private $translationNamespace;
+
+    public function __construct(IlluminateTranslatorInterface $decorated, string $translationNamespace)
     {
         $this->decorated = $decorated;
+        $this->translationNamespace = $translationNamespace;
     }
 
     /**
+     * @param string $message
      * @param mixed[] $parameters
+     *
+     * @return string
      */
     public function trans(string $message, array $parameters): string
     {
         // TODO: rework after upgrading all the illuminate and laravel packages to ^6.0
         $method = \method_exists($this->decorated, 'trans') ? 'trans' : 'get';
 
+        $translation = $this->decorated->{$method}(
+            \implode([$this->translationNamespace, '::', \trim($message)]),
```

Comment:
Use `\sprintf()` instead of `\implode()` array are slow and really complex for this use case.

Reply:
Thanks, I'll use `sprintf` instead